XamarinCommunityToolkit.UnitTests/ObjectModel/AsyncCommand_Tests.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xamarin.CommunityToolkit.ObjectModel;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests.ObjectModel
+{
+	public class AsyncCommand_Tests
+	{
+		[Fact]
+		public void Constructor_NullExecute()
+		{
+			// Assert
+			Assert.Throws<ArgumentNullException>(() => new AsyncCommand(null));
+			Assert.Throws<ArgumentNullException>(() => new AsyncCommand<object>(null));
+		}
+
+		[Fact]
```

Comment:
Just a proposition to convert some of the test-cases to "Theory".

For example, this one is a good candidate.